packages/web-components/fast-element/src/components/element-controller.ts

```diff
@@ -113,37 +115,36 @@ export class Controller<
     }
 
     /**
-     * Gets/sets the primary styles used for the component.
-     * @remarks
-     * This value can only be accurately read after connect but can be set at any time.
+     * The main set of styles used for the component, independent
+     * of any behavior-specific styles.
```

Comment:
Are you using "behavior" here to reference `HostBehavior`? If so, this comment might be a little misleading because anything can add styles to the controller. Maybe this should open up to "dynamically applied styles" or something similar?